src/renderer/services/file-management-system/index.ts

```diff
@@ -157,29 +169,18 @@ export default class FileManagementSystem {
    */
   public async upload(
     upload: UploadJob,
-    onProgressInfo: (progress: UploadProgressInfo) => void
   ): Promise<void> {
     try {
-      const [fssStatus, source, fileSize] = await this.register(upload);
-      const onProgressBytes = (bytesUploaded: number) => onProgressInfo({ bytesUploaded, totalBytes: fileSize })
-      if (upload.serviceFields.localNasShortcut) {
-        await this.waitForServerCopy(fssStatus.uploadId, onProgressBytes);
-      } else if(fssStatus.chunkStatuses && fssStatus.chunkStatuses[0]) {  //Handles the case where FUA believes this is a new upload, 
-                                                                          //but actually, it is partially complete already.
-
-        const onRetryProgress = (                                         // Update the JSS FUA upload job with the FSS upload ID field
-          uploadId: string,
-          progress: UploadProgressInfo
-        ) => {
-          onProgressInfo(progress)
-        };
-        await this.retry(upload.jobId, onRetryProgress);                  // create the FUA JSS upload id and info here                  
-      } else {
+      const [fssStatus, source] = await this.register(upload);
+      if(fssStatus.chunkStatuses && fssStatus.chunkStatuses[0]) {  
+        //Handles the case where FUA believes this is a new upload, 
+        //but actually, it is partially complete already.
+        await this.retry(upload.jobId);                  // create the FUA JSS upload id and info here                  
```

Comment:
I'm confused by this comment, the FUA JSS upload ID has to exist before this method is called.

Reply:
Removed.
I think that was a merge artifact.  this code came from a parallel branch that @kmitcham worked on.